_templates/layout.html

```diff
@@ -0,0 +1,31 @@
+{% extends '!layout.html' %}
+
+{% block footer %}
+
+<div class="footer">
+    <p>© Copyright <a href="https://github.com/publiccodeyml/publiccode.yml/blob/main/AUTHORS.md">publiccode.yml authors</a> <span style="margin-left:8px; margin-right: 8px;">|</span> <a href="https://github.com/publiccodeyml/publiccode.yml">GitHub</a></p>
```

Comment:
Wondering if it makes sense to add this copyright line when the standard is CC0 public domain.

Maybe we can just mention CC-0 in the footer with a link to the license. Wdyt?